addon/libconiks.so

```diff
@@ -0,0 +1 @@
+../chrome/content/libconiks.so
```

Comment:
> Is this possible?

I think so.  golang should be available in the vm.

After the `install` step (so, in `before_script`), you'd do something like,

```
go get github.con/coniks-sys/coniks-go  // or just clone it if it's simpler
cd /path/to/coniks-go/client/cgo
make
cp .so /addon
```

But don't spend too much time on it.  You can commit to .so and we can fix it up later.